common/peerprovider/ringpopprovider/config.go

```diff
@@ -64,7 +64,9 @@ const (
 // Config contains the ringpop config items
 type Config struct {
 	// Name to be used in ringpop advertisement
-	Name string `yaml:"name" validate:"nonzero"`
+	Name string `yaml:"name"`
+	// Static Address to be announced for other nodes
+	Address string `yaml:"address"`
```

Comment:
@mantas-sidlauskas I'm cleaning up old open PRs. Are you planning to continue on this anytime soon? If not let's close it and open a new one when you will be actively working on this improvement.